nextjs-metrics-viewer/app/api/analytics/services/validate_data.ts

```
@@ -0,0 +1,73 @@
+      if (!this.validateHeaders(headers, expectedHeaders)) {
+        return {
+          success: false,
+          error: `Encabezados inválidos. Se esperaban: ${expectedHeaders.join(
+            ", ",
+          )}`,
+        };
+      }
+
+      const data: AnalyticsModel[] = parsedCsv.data.map((row: any) => {
+        const numericValue = parseFloat(row.Value);
+        if (isNaN(numericValue)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)